js/src/components/ParagraphHighlightFullWidth.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import Box from '@material-ui/core/Box';
+import Typography from '@material-ui/core/Typography';
+import LessonGrid from '@anu/components/LessonGrid';
+
+const useHighlightStyles = makeStyles((theme) => ({
+  root: {
+    paddingTop: theme.spacing(2),
+    paddingBottom: theme.spacing(2),
+    backgroundColor: (props) => theme.palette.paragraphHighlight[props.color],
+    [theme.breakpoints.up('sm')]: {
+      paddingTop: theme.spacing(6),
+      paddingBottom: theme.spacing(6),
+    },
+  },
+}));
+
+const useHeadingStyles = makeStyles((theme) => ({
+  heading: {
+    fontWeight: theme.typography.fontWeightBold,
+    fontSize: '1.25rem',
+  },
+}));
+
+const useBodyStyles = makeStyles((theme) => ({
+  text: {
+    fontWeight: theme.typography.fontWeightMedium,
+    fontSize: '1rem',
+    [theme.breakpoints.up('sm')]: {
+      fontSize: '1.5rem',
+      lineHeight: '2.063rem',
+    },
+    '& > p': {
+      marginBottom: theme.spacing(4),
+    },
+    '& > p:first-child': {
+      marginTop: 0,
+    },
+    '& > p:last-child': {
+      marginBottom: 0,
+    },
+  },
+}));
+
+const ParagraphHighlightFullWidth = ({ title, text, color }) => {
+  const highlightClasses = useHighlightStyles({ color: color });
```

Comment:
You didn't have to create each style separately. It could have been something a bit more simple like this https://material-ui.com/styles/advanced/#makestyles. No need to change that now - not a big deal, it would just keep the code more consistent with other components.

Reply:
true, i'll fix this